File: 1. Begginer/12- Pida al usuario un número entero, diga si es par o impar,y repita todo lo anterior hasta que el número que se introduzca sea 0. (Nota para saber si un número es par, se mira si el resto de s/main.php
<?php 

for ($i = 0; $i < 2; $i++) {
	$num = (int) readline("Ingrese un número: ");
	if ($num != 0) {
		if ($num%2 == 0) {
			echo "El número es par \n";
		} else {
			echo "El número es impar \n";
		}
		$i = 0;
	} else {
		echo "El programa ha finalizado";
		break;
	}
}

File: 1. Begginer/12- Pida al usuario un número entero, diga si es par o impar,y repita todo lo anterior hasta que el número que se introduzca sea 0. (Nota para saber si un número es par, se mira si el resto de s/main.cpp
#include <iostream>

using namespace std;

int main(void) {
	for (int i = 0; i < 1; i) {
		int num;
		cout << "Ingrese un número: ";
		cin >> num;
		if (num != 0) {
			if (num%2 == 0) {
				cout << "El número es par\n";
			} else {
				cout << "El número es impar\n";
			}
		} else {
			cout << "El programa ha finalizado";
			i = 1;
		}
	}
}